docker-compose.local.yml

```
@@ -0,0 +1,83 @@
+      - ./jsconfig.json:/tourRanking/jsconfig.json:ro
+      - ./tourrankings-google-service-account-key.json:/tourRanking/tourrankings-google-service-account-key.json:ro
+      # Preserve container's node_modules
+      - /tourRanking/node_modules
+    environment:
+      # These override/supplement .env.local
+      NODE_ENV: development
+      PORT: 8080
+      DATA_DIR: /tourRanking/data/csv
+      LOG_LEVEL: debug
+      PUPPETEER_HEADLESS: "true"
+      LOG_LEVEL: debug
```

YAMLlint (1.37.1)

[error] duplication of key "LOG_LEVEL" in mapping

(key-duplicates)